home_work_7/index.js

```diff
@@ -0,0 +1,192 @@
+    driverBirthYear: validateValue(driverBirthYear) || EMPTY_VALUE,
+    carBrand: validateValue(carBrand) || EMPTY_VALUE,
+    sponsor: validateValue(sponsor) || EMPTY_VALUE,
+    driverHasAccess: Boolean(driverHasAccess),
+  };
+
+  return NEW_TEAM;
+};
+
+createAndRegisterNewTeam();
+// console.log(createAndRegisterNewTeam("Spirit", "John", 1990, "BMW", "HDRezka"));
+
+/*
+2. Запитати дані (через prompt та циклічно) про нового користувача у системі та зберегти його в об'єкт.
+Дані про користувача - логін, пароль, місто, країна, стать, вік
+*/
+
+const createNewUser = () => {
+  const USER = {
+    login: "",
+    password: "",
+    city: "",
+    country: "",
+    sex: "",
+    age: "",
+  };
+
+  for (const [key] of Object.entries(USER)) {
+    let result = prompt([key]);
+    if (result === null || result === "") {
+      USER[key] = EMPTY_VALUE;
+    } else {
+      USER[key] = result;
+    }
+  }
+  return USER;
+};
+const newUser = createNewUser();
+// console.log(newUser);
+
+/*
+3. Створити функцію, яка буде міняти дані в конкретного користувача
+створеного пунктом вище. Наприклад сhangeUserData(user_1, city, 'Kherson).
+Де user_1 - обʼєкт в якому буде мінятись, city - поле,
+яке буде мінятися на нове значення - 'Kherson'
+*/
+
+const editUserData = (object, key, value) => {
+  return (object[key] = value);
+};
+
+editUserData(newUser, "city", "Chuguiv");
+// console.log(newUser);
+
+/*
+4. Створити об'єкт cтудента - name, surname, age, course, city, greeting, addHomework. greeting - метод,
+котрий виводить повідомлення через console.log('Hi. everyone!').
+addHomework - метод, котрий виводить повідомлення
+через console.log('Sending my howework... Please, wait')
+*/
+
+const STUDENT = {
+  name: "Viacheslav",
+  surname: "Sydorenko",
+  age: 29,
+  course: 1,
+  city: "Dnipro",
+  greeting: function () {
+    return console.log("Hi. everyone!");
+  },
+  addHomework: function () {
+    return console.log("Sending my howework... Please, wait");
+  },
+};
+STUDENT.greeting();
+STUDENT.addHomework();
+
+/*
+5. Cтворити функцію isEmpty, яка повертає true,
+якщо об’єкт не має властивостей (порожній), інакше false
+*/
+
+function isEmpty(value) {
+  if (!validateValue(value) || typeof value !== "object") {
+    return "false value";
+  }
+  if (Object.keys(value).length === 0) {
+    return "true";
+  } else {
+    return "false";
+  }
+}
+isEmpty();
+//const testObj = {};
+// console.log(isEmpty());
+
+/*
+6. Створити об’єкт для зберігання виручки команди продавців, наприклад:
+{Taras: 2000, Marta: 10 Ivan: 1200, Petro: 400, Roma: 366, Alina: 829}
+*/
+
+const SALES_TEAM_REVENUE = {
+  Marta: 1000,
+  Alina: 2000,
+  Petro: 300,
+  Taras: 4000,
+  Roma: 800,
+  Ivan: 5000,
+};
+
+/*
+7. Створити фукнцію, яка працює з цим обʼєктом та
+обчислює суму всіх виручок та виводить результат через сonsole.log
+*/
+
+const getSumSalesmensRevenue = (salesmen) => {
+  let sumRevenue = 0;
+  for (let Revenue of Object.values(salesmen)) {
```

Comment:
No need to capitalize here

```suggestion
  for (let revenue of Object.values(salesmen)) {
```